src/libraries/System.Private.Uri/src/System/UriExt.cs

```diff
@@ -81,9 +81,7 @@ private void InitializeUri(ParsingError err, UriKind uriKind, out UriFormatExcep
 
             bool hasUnicode = false;
 
-            _iriParsing = IriParsingStatic(_syntax);
```

Comment:
It looks like there are several references to this in the function... would it be better to access it once and store it into a local?  I know you said it's cheap... is it cheap enough that a local isn't helpful?